test/integration/sequelize.test.js

```diff
@@ -647,8 +647,27 @@ describe(Support.getTestDialectTeaser('Sequelize'), function() {
     });
   });
 
+  describe('truncate', function() {
+    it("truncates all models", function() {
+      var Project = this.sequelize.define('project' + config.rand(), { title: DataTypes.STRING });
+
+      return this.sequelize.sync({ force: true }).then(function() {
+        return Project.create({ title: 'bla' });
```

Comment:
Puh... I thought about something like that while writing this but I hoped you didn't mind me being lazy :D